.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: go
+go_import_path: github.com/coreos/dex
```

Comment:
please remove this, it doesn't have anything to do with the change.

Reply:
Removed. I added go_import_path in travis.yaml because I was not able to run tests for my fork in Travis. With go_import_path Travis should work for your project as well as for all forks